### DISCUSSION TITLE
Feedback symbols' meaning

### DISCUSSION THREAD

**Discussion by nigoshh** [OpenMined/courses#260] [Feedback] [answered]
Some meta-feedback about the feedback system itself :grinning: I don't understand the difference in meaning between the two symbols :ok_hand: and :+1:. Don't they both indicate liking the content, i.e. positive feedback? Or is one of them "more positive" than the other? In that case I'd guess that :+1: is more positive, just based on the order; but it's unclear when we should use one and when the other.

**Comment by Kiaka007** [comment, marked as answer]
Great feedback I'll add this as an issue in GitHub.
Smileys might be a better alternative maybe a ☹😐🙂. They represent:

- 👎Dislike or Bad
- 👌 Ok or Neutral
- 👍 Like or Good

**Comment by Kiaka007** [comment]
Now captured in [Issue: 417](https://github.com/OpenMined/courses/issues/417)
